tests/plugins/widget/manual/contextmenushortcuts.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.11.4, bug, 1901, widget, accessibility
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, wysiwygarea, contextmenu, placeholder
+
+## Keyboard shortcuts for widgets' context menu
+
+### Scenario 1: `SHIFT + F10`
+
+1. Focus widget in editor.
+1. Press `SHIFT + F10`. **Expected:** Custom CKEditor context menu opens.
+
+### Scenario 2: `CTRL + SHIFT + F10`
+
+1. Focus widget in editor.
+1. Press `CTRL + SHIFT + F10`. **Expected:** Custom CKEditor context menu opens.
```

Comment:
It should be enough to say that "Context menu opens".

Reply:
I am not sure about this modification because if CKEditor's custom menu will not work and default browser menu opens instead, IMO the test should not pass, while with shorter instruction note I would let it pass. If you think it's not about to happen then of course we can go with simpler version.

Reply:
It's about the naming convention, mostly you will see just `context menu` referencing to our context menu plugin. I'm not sure if we have any reference to the browser native context menu. Does it resolve your issue?

Reply:
If in all tests we reference to custom context menu then you're right that more specific clarifications in this case are not necessary.